Code/src/main/resources/ui/style.css

```diff
@@ -65,14 +65,21 @@ CustomUIPane {
     -fx-text-alignment: center;
 }
 
-.display.block .argumentType {
-    -fx-padding: 0 5 0 5;
+.display.block .argumentType, .value.block .argumentType {
```

Comment:
I'd say this is not an argument type (black-on-white) but a result type (white-on-black).